ftw/simplelayout/browser/client/dist/simplelayout.js

```diff
@@ -635,6 +635,13 @@ define('app/simplelayout/Block',[
       return this;
     };
 
+    this.markLinks = function() {
```

Comment:
The `simplelayout.js` is a generated file. The original function is in `Block.js`.
@bierik recommended this structure. I think it makes sense. `Block.js` for the implementation and `main.js` for the integration.
It is also way easier to test this way.